src/migrations/1679669193721-migrations.ts

```diff
@@ -0,0 +1,1612 @@
+    // rename all tables to old
+    await queryRunner.query('ALTER TABLE action RENAME TO action_old');
+    await queryRunner.query(
+      'ALTER TABLE action_request_export RENAME TO action_request_export_old',
+    );
+    // don't use we never used admin_role anymore
+    await queryRunner.query('DROP TABLE IF EXISTS admin_role');
+    await queryRunner.query('ALTER TABLE app RENAME TO app_old');
+    await queryRunner.query('ALTER TABLE app_data RENAME TO app_data_old');
+    await queryRunner.query('ALTER TABLE app_action RENAME TO app_action_old');
+    await queryRunner.query('ALTER TABLE app_setting RENAME TO app_setting_old');
+    await queryRunner.query('ALTER TABLE category RENAME TO category_old');
+    await queryRunner.query('ALTER TABLE category_type RENAME TO category_type_old');
+    await queryRunner.query('ALTER TABLE chat_mention RENAME TO chat_mention_old');
+    await queryRunner.query('ALTER TABLE chat_message RENAME TO chat_message_old');
+    await queryRunner.query('ALTER TABLE flag RENAME TO flag_old');
+    await queryRunner.query('ALTER TABLE invitation RENAME TO invitation_old');
+    await queryRunner.query('ALTER TABLE item RENAME TO item_old');
+    await queryRunner.query('ALTER TABLE item_category RENAME TO item_category_old');
+    await queryRunner.query('ALTER TABLE item_flag RENAME TO item_flag_old');
+    await queryRunner.query('ALTER TABLE item_like RENAME TO item_like_old');
+    await queryRunner.query('ALTER TABLE item_member_login RENAME TO item_member_login_old');
+    await queryRunner.query('ALTER TABLE item_membership RENAME TO item_membership_old');
+    await queryRunner.query('ALTER TABLE item_tag RENAME TO item_tag_old');
+    await queryRunner.query('ALTER TABLE item_validation RENAME TO item_validation_old');
+    await queryRunner.query(
+      'ALTER TABLE item_validation_group RENAME TO item_validation_group_old',
+    );
+    await queryRunner.query(
+      'ALTER TABLE item_validation_review RENAME TO item_validation_review_old',
+    );
+    await queryRunner.query(
+      'ALTER TABLE item_validation_process RENAME TO item_validation_process_old',
+    );
+    await queryRunner.query(
+      'ALTER TABLE item_validation_status RENAME TO item_validation_status_old',
+    );
+    await queryRunner.query(
+      'ALTER TABLE item_validation_review_status RENAME TO item_validation_review_status_old',
+    );
+    await queryRunner.query('ALTER TABLE member RENAME TO member_old');
+    // don't use permission (admin?)
+    await queryRunner.query('DROP TABLE IF EXISTS permission');
+    await queryRunner.query('ALTER TABLE publisher RENAME TO publisher_old');
+    await queryRunner.query('ALTER TABLE recycled_item RENAME TO recycled_item_old');
+    // don't use role and role_permission (admin)
+    await queryRunner.query('DROP TABLE IF EXISTS role_permission');
+    await queryRunner.query('ALTER TABLE tag RENAME TO tag_old');
```

Comment:
```suggestion
    // don't use role and role_permission (admin)
    await queryRunner.query('DROP TABLE IF EXISTS role_permission');
    await queryRunner.query('ALTER TABLE tag RENAME TO tag_old');
    // don't use permission (admin?)
    await queryRunner.query('DROP TABLE IF EXISTS permission');
    await queryRunner.query('ALTER TABLE publisher RENAME TO publisher_old');
    await queryRunner.query('ALTER TABLE recycled_item RENAME TO recycled_item_old');
```

role_permission references permissions so it must be deleted first